coalib/bears/Bear.py

```diff
@@ -16,14 +16,31 @@
 from coalib.bears.BEAR_KIND import BEAR_KIND
 from coalib.output.printers.LogPrinter import LogPrinterMixin
 from coalib.results.Result import Result
+from coalib.misc.Constants import FALSE_STRINGS, TRUE_STRINGS
 from coalib.results.TextPosition import ZeroOffsetError
 from coalib.settings.FunctionMetadata import FunctionMetadata
 from coalib.settings.Section import Section
 from coalib.settings.ConfigurationGathering import get_config_directory
+from coalib.settings.Setting import typed_list
 
 from .meta import bearclass
 
 
+def _debugger_helper(bear):
+    if bear is None:
+        return False
+    elif 'debug_bears' not in bear.section or (
+            bear.section['debug_bears'].value.lower() in FALSE_STRINGS):
+        return False
+    elif bear.name.lower() in (
```

Comment:
you do not need `elif` if the previous branch did a `return`.

avoiding the `elif` means you can also do some computation and use variables *before* the next `if`, instead of putting so much unreadable stuff into an `if` clause like you have done here